server/worker_pool.go

```diff
@@ -80,6 +87,7 @@ func newWorkerPool(s *Server) (*workerPool, error) {
 				serverUtilization.WithLabelValues("rsa").Set(float64(wp.RSA.Busy()) / float64(s.config.rsaWorkers))
 				serverUtilization.WithLabelValues("ecdsa").Set(float64(wp.ECDSA.Busy()) / float64(s.config.ecdsaWorkers))
 				serverUtilization.WithLabelValues("other").Set(float64(wp.Other.Busy()) / float64(s.config.otherWorkers))
+				serverUtilization.WithLabelValues("remote").Set(float64(wp.Remote.Busy()) / float64(s.config.remoteWorkers))
```

Comment:
I can't add a comment on Line 78, but `for _, label := range []string{"rsa", "ecdsa", "other", "limited"} ` should also have `remote`

Reply:
Same with line 100, `wp.Remote.Destroy()` might be necessary, though we somehow don't destroy the RSA pool?